docs/reference/project/functions.mdx

```diff
@@ -1 +1,28 @@
-TODO...
+A Nitric function is described by its definition in a `nitric.yaml` file. Nitric functions are a type of [Service](../../intro/services) that offers a portable way to develop function-as-a-service (FaaS) applications, which can then be deployed to
+popular Serverless offerings like AWS Lamda, Google CloudRun and Azure AppService.
+
+## Properties
+
+| Property | Value      | Required | Description                                                                                                                                     |
+| -------- | ---------- | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------- |
+| name     | `string`   | Yes      | The name of the Nitric function. Must be unique to the stack file as it is used as the reference to the function for entrypoints and apis.      |
```

Comment:
Looks good, one more catch with `name` as a property, is it isn't really a property of a function and more part of it's general definition, as it's just a key that a function object is stored under. I've been trying to think of a way to represent this and listing it as a property could work. Just a note that it's not in other property tables.

Reply:
I'll remove it for consistency amongst the reference docs.